wrangler-core/src/main/java/io/cdap/directives/transformation/Decode.java

```diff
@@ -88,8 +88,8 @@ public void initialize(Arguments args) throws DirectiveParseException {
     type = type.toUpperCase();
     if (!type.equals("BASE64") && !type.equals("BASE32") && !type.equals("HEX")) {
       throw new DirectiveParseException(
-        String.format("Type of decoding specified '%s' is not supported. Supports base64, base32 & hex.",
-                      type)
+        String.format("Error encountered while initializing '%s' : Type of encoding specified '%s' is not supported. " +
```

Comment:
Encoding type '%s' is not supported. Supported types are ...

Reply:
I think it should be `Decoding type...`